pkg/drivers/vfkit/vfkit.go

```diff
@@ -152,6 +155,24 @@ func (d *Driver) GetState() (state.State, error) {
 	return state.Running, nil
 }
 
+func (d *Driver) getVmnetHelperState() (state.State, error) {
+	if d.VmnetHelper == nil {
+		return state.Stopped, nil
+	}
+	return d.VmnetHelper.GetState()
+}
+
+// GetState returns driver state. Since vfkit driver may use 2 processes
+// (vmnet-helper, vfkit), this returns combined state of both processes.
+func (d *Driver) GetState() (state.State, error) {
+	if vfkitState, err := d.getVfkitState(); err != nil {
+		return state.Error, err
+	} else if vfkitState == state.Running {
+		return state.Running, nil
```

Comment:
if `getVfkitState` returns `running`, is it guaranteed that vmnet will also be running? or should it be `s/Running/Stopped` in this block?

Reply:
Yes, the semantics is returning Running if at least one process is running, since we used it only in stop or delete flows. If some process is running, we need to continue polling and maybe retry the stop or kill.

Reply:
If it's not used in `Start` to wait until the state is `running`, then this should be fine. My worry was if something expects the VM to be running and usable when only one of the 2 processes is up.

Reply:
We may need to improve this later if we want to use the state for checking if start finished. Currently Start() is blocking and when it returns we have 2 processes running. (actually 3: vfkit, sudo, vmnet-helper).